sos/plugins/openstack_tripleo.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2019 Red Hat, Inc., Cedric Jeanneret <cjeanner@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+
+
+class OpenStackTripleO(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
+    """Log installation informations from OpenStack Installer
+    """
+
+    plugin_name = 'openstack_tripleo'
+
```

Comment:
When the plugin will be enabled (so far it would be *any* time)? What installed package, kernel module loaded, file present or running service should enable the plugin?

Shall not be the plugin added to a profile? Cf. https://github.com/sosreport/sos/blob/master/sos/plugins/openstack_nova.py#L24 (and https://github.com/sosreport/sos/blob/master/sos/plugins/openstack_nova.py#L208 for enablement of the plugin).

Reply:
good remark - will take care of that in the next update. But I will probably need to make another PR "before" this one in order to inject a new method for secret handling.

Reply:
No need for another PR. Just modify your branch and forcefully push it to github.

Reply:
oh, yeah, I was more thinking about a "generic method in order to mask secrets" (see previous comment with @bmr-cymru ) - but that one isn't useful. I'll push shortly the change adding profiles and packages.